.github/workflows/workflow-on-branch-delete.yml

```diff
@@ -0,0 +1,20 @@
+name: Labs delete
+on: delete
+env:
+  APP: spinnsyn-frontend
+  branchname: ${{ github.ref_name }}
+jobs:
+  deploy-branch-to-labs-gcp:
+    if: startsWith(github.ref, 'refs/heads/labs-')
+    name: Delete branch labs-gcp
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - run: echo "IMAGE=docker.pkg.github.com/${GITHUB_REPOSITORY}/$APP:latest" >> $GITHUB_ENV
```

Comment:
lurer litt på om latest kan være litt flytende. At den er ikke garantert å bli den nyeste latest.

Reply:
Ja, tror vi må ha med image for at den skal få lov å deploye
Så tenker den kan hente latest og sette replicas til 0, så slipper vi ekstra kode for å bygge og push ny versjon, eller finne igjen det gamle versjonsnummeret

Reply:
enig, eneste ulempen er hvis noen faktisk tester på branch versjonen igjen. Vi kunne alternativt deploya en statisk side som sier. "Hei, jeg er sletta :)" . Vi kan mekke noe sånn på sikt